packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/cli.integtest.ts

```diff
@@ -1,12 +1,9 @@
 import { promises as fs, existsSync } from 'fs';
 import * as os from 'os';
 import * as path from 'path';
-import { retry, sleep } from '../helpers/aws';
-import { cloneDirectory, MAJOR_VERSION, shell, withDefaultFixture } from '../helpers/cdk';
-import { randomInteger, withSamIntegrationFixture } from '../helpers/sam';
-import { integTest } from '../helpers/test-helpers';
+import { integTest, cloneDirectory, shell, withDefaultFixture, retry, sleep, randomInteger, withSamIntegrationFixture, RESOURCES_DIR } from '../../lib';
 
-jest.setTimeout(600_000);
+jest.setTimeout(60 * 60_000); // Includes the time to acquire locks
```

Comment:
[non-blocking] If something starts to edge-case, how can we quickly tell we're hitting that scenario?

Reply:
We'll see timeouts